### ISSUE TITLE
Questions about train.log 

### ISSUE THREAD

**Issue by andrewsx** [wang3702/EnAET#3]
Hello,

I was wondering in the train.log what loss, Closs, Eloss, wd, waet and top1avg mean?

Thanks for any clarification!
Warm regards,
Andrew

**Comment by wang3702** [comment]
Hi,
Thanks for your question! loss is the total loss as described in the paper, which is the sum of mixmatch_loss+aet_loss+kl_loss. Here Closs is the cross-entropy loss in MixMatch loss. Eloss is another term in MixMatch loss, which is a MSE loss as described in MixMatch paper. wd is the weight for the Eloss. waet is the weight for aet-loss1(projective transformation), other weights for aet loss also follow the same trend to warm up. top1avg is the accuracy when I used the average predictions of all different transformation.
Any questions, welcome to update in this issue. If your question is resolved, please close the issue. Thanks!
